prototypes/user-preferences/src/App.js

```diff
@@ -0,0 +1,155 @@
+import "./App.css";
+import { Card } from "primereact/card";
+import { Panel } from "primereact/panel";
+//import { Splitter, SplitterPanel } from 'primereact/splitter';
+import React, { useState, useLayoutEffect } from "react";
+import { InputNumber } from "primereact/inputnumber";
+import { Dropdown } from "primereact/dropdown";
+import { AutoComplete } from "primereact/autocomplete";
+import "primereact/resources/themes/lara-dark-purple/theme.css";
+
+function App() {
+  const [value1, setValue1] = useState(60);
+  const [value2, setValue2] = useState(20);
+  const [value3, setValue3] = useState(1600);
+  const [value4, setValue4] = useState(150);
+  const [value5, setValue5] = useState(180);
+
+  const [selectedRestriction, setSelectedRestriction] = useState(null);
+  const restrictions = [
+    { name: "gluten free", code: "GF" },
+    { name: "dairy free", code: "DF" },
+    { name: "nut free", code: "NF" },
+  ];
+
+  return (
+    <div className="App">
+      <header className="App-header">
+        <Panel header="Macro Goals">
+          <p className="m-0">
+            <label>Protein:</label>{" "}
+            <InputNumber
+              inputId="stacked-buttons"
+              value={value1}
+              onValueChange={(e) => setValue1(e.value)}
+              inputClassName="quantity-input"
+            />{" "}
+            grams
+          </p>
+          <p className="m-0">
+            <label>Carbs:</label>{" "}
+            <InputNumber
+              inputId="stacked-buttons"
+              value={value2}
+              onValueChange={(e) => setValue2(e.value)}
+              inputClassName="quantity-input"
+            />{" "}
+            grams
+          </p>
+          <p className="m-0">
+            <label>Fat:</label>{" "}
+            <InputNumber
+              inputId="stacked-buttons"
+              value={value1}
+              onValueChange={(e) => setValue1(e.value)}
+              inputClassName="quantity-input"
+            />{" "}
+            grams
+          </p>
+          <p className="m-0">
+            <label>Calories:</label>{" "}
+            <InputNumber
+              inputId="stacked-buttons"
+              value={value3}
+              onValueChange={(e) => setValue3(e.value)}
+              inputClassName="quantity-input"
+            />{" "}
+            kcals
```

Comment:
```suggestion
            />
            <span> kcals</span>
```

Reply:
perfect, i was thinking my way was a goofy way to render a space